spec/jobs/socure_shadow_mode_proofing_job_spec.rb

```diff
@@ -18,7 +18,7 @@
   end
 
   let(:applicant_pii) do
-    Idp::Constants::MOCK_IDV_APPLICANT_WITH_PHONE
+    Idp::Constants::MOCK_IDV_APPLICANT_WITH_SSN
```

Comment:
Maybe one more test in this file asserting that phone_source makes it into the logged event